frontend/src/concepts/pipelines/content/tables/pipelineRun/PipelineRunTable.tsx

```diff
@@ -201,6 +230,21 @@ const PipelineRunTable: React.FC<PipelineRunTableProps> = ({
               primaryToolbarAction,
               ...(compareRunsAction ? [compareRunsAction] : []),
               toolbarDropdownAction,
+              isExperimentsEnabled
+                ? [
+                    <Tooltip key="custom-metrics-columns" content="Customize metrics columns">
```

Comment:
tooltip wont show on disabled buttons. you need to use the ariadisabled prop instead

Reply:
also a reason why it is disabled might help too

Reply:
I added some different messages. 

Loading state: `Customize metrics columns: Loading metrics...`
Page loads with runs without metrics available: `Customize metrics columns: No metrics available`

Let me know what you think. Maybe @kaedward could chime in on this as well.